fs/fs.go

```diff
@@ -282,7 +259,7 @@ func (self *RealFsInfo) GetDirFsDevice(dir string) (*DeviceInfo, error) {
 }
 
 func (self *RealFsInfo) GetDirUsage(dir string) (uint64, error) {
-	out, err := exec.Command("du", "-s", dir).CombinedOutput()
+	out, err := exec.Command("/bin/sh", "-c", "ulimit -t 1; du -s", dir).CombinedOutput()
```

Comment:
+1. I don't know if it's useful to set a hard 1-second limit as @jimmidyson mentioned.